zebra-consensus/src/transaction.rs

```diff
@@ -575,12 +575,10 @@ where
                     miner_fee,
                     legacy_sigop_count,
                 },
-                Request::Mempool { transaction, .. } => {
+                Request::Mempool { transaction: ref tx, .. } => {
                     let transaction = VerifiedUnminedTx::new(
-                        transaction,
-                        miner_fee.expect(
-                            "unexpected mempool coinbase transaction: should have already rejected",
-                        ),
+                        tx.clone(),
```

Comment:
Why this change?

Reply:
Thanks for spotting this. It was required for the conditional compilation I had earlier, which is gone now. I'll revert it.